### DISCUSSION TITLE
Support CIS Kubernetes Benchmark 1.6.0

### DISCUSSION THREAD

**Discussion by mozillazg** [aquasecurity/kube-bench#664] [Development]
How about support the latest CIS Kubernetes Benchmark 1.6.0?

**Comment by mozillazg** [comment]
The CIS 1.6.0 no longer use the [scoring status], it will use [assessment status] instead, and it will use [Automated]/[Manual] instead of [Scored]/[Not Scored] for each checks.

**Reply by lizrice** [reply]
We can use our existing `type: manual` support to indicate the tests that need to be run manually, these would output `[WARN]` to indicate that the user has to do something for these tests. 

In kube-bench we have options `--scored` and `--unscored` that allow users to specify whether to run those tests (they both default to true so we run all the tests). I wonder whether people will want a similar option so that they can choose to only run the automated tests? If so, one hacky workaround would be to continue to mark tests as `scored: true` for automated tests and `scored: false` for the manual ones.

**Reply by yoavrotems** [reply]
To my opinion maybe this change of names [Automated]/[Manual] instead of [Scored]/[Not Scored] means we should just change the tests titles and leave the `scored: true/false` the same, and maybe change the kube-bench behavior that automatically runs both scored and not scored tests to run only scored tests (now could Automated) and if the user choose to run the manual he should run with the not-scored flag (maybe should change the name of the flag as well)

**Comment by lizrice** [comment]
I've raised an [issue](https://github.com/aquasecurity/kube-bench/issues/666) to cover this enhancement

**Comment by lizrice** [comment]
#678 is now merged, and we'll tag a new release with this shortly

**Comment by lizrice** [comment]
Finally, we have cut https://github.com/aquasecurity/kube-bench/releases/tag/v0.4.0 🎉
